### DISCUSSION TITLE
Feature request: Return miner's committed transactions root on eth_sendBundle

### DISCUSSION THREAD

**Discussion by sequence-dev-1** [flashbots/pm#70] [searchers]
One of the reasons a Flashbots bundle may not be included on-chain is because of latency. 

In the case where a miner has already formed block that has been sent out to its pool of hash power, then even if a bundle has the winning score it will have to wait for the commit time to expire and the miner to form a new block.

Therefore a useful piece of information is the transactions root hash at the time the miner received `eth_sendBundle`. If the previously committed [transactions root hash](https://github.com/ethereum/go-ethereum/blob/master/core/types/block.go#L74) is returned and it is the same that made it on-chain, then there is a very high likelihood the bundle was not included due to latency (assuming the bundle submitted did have a higher score).

**Comment by thegostep** [comment]
How would this handle each miner having a different transaction root?

**Reply by sequence-dev-1** [reply]
I think that's the biggest question right now. Perhaps the relay could aggregate the responses from each miner and return an array of transaction roots?

**Reply by thegostep** [reply]
Are there other ways to provide information about latency without this approach?
